src/scale.js

```diff
@@ -0,0 +1,63 @@
+import {isFinite, valueOrDefault} from 'chart.js/helpers';
+
+export function adjustScaleRange(chart, scale, annotations) {
+  const range = getScaleLimits(scale, annotations);
+  let changed = changeScaleLimit(scale, range, 'min', 'suggestedMin') || changeScaleLimit(scale, range, 'max', 'suggestedMax');
```

Comment:
The later function is not called when first one returns true. So this does not work properly.

I guess we don't have any tests that would catch this.

Reply:
move to another PR